test/stress/logs/agent_config.json

```diff
@@ -0,0 +1,53 @@
+{
+    "agent": {
+      "debug": true
+    },
+    "metrics": {
+      "namespace": "CloudWatchAgentStress",
+      "metrics_collected": {
+        "net": {
+          "resources": [
+            "eth0"
+          ],
+          "measurement": [
+            "bytes_sent",
+            "packets_sent"
+          ],
+          "metrics_collection_interval": 1
+        },
+        "procstat": [
+          {
+            "exe": "cloudwatch-agent",
+            "measurement": [
+              "cpu_usage",
+              "memory_rss",
+              "memory_swap",
+              "memory_vms",
+              "memory_data",
+              "num_fds"
+            ],
+            "metrics_collection_interval": 1
+          }
+        ]
+      },
+      "append_dimensions": {
+        "InstanceId": "${aws:InstanceId}"
+      },
+      "force_flush_interval": 10
```

Comment:
Any reason this is needed here but not for the other one?

Reply:
Yes. If we increase it more further, it will not constraint the resources when submitting metrics and avoid sample count being reduced to many. That's one of the reason why I have to reduce to 5 sample count but want to near our default force flush interval